src/main/groovy/org/jetbrains/intellij/PrepareSandboxTask.groovy

```diff
@@ -118,7 +123,7 @@ class PrepareSandboxTask extends Sync {
     }
 
     private void configureClasses(@NotNull IntelliJPluginExtension extension, boolean inTests) {
-        destinationDir = new File(Utils.pluginsDir(extension, inTests), "$extension.pluginName")
+        destinationDir = new File(Utils.pluginsDir(extension, inTests))
```

Comment:
This means that all plugins will be wiped on `cleanPrepareSandbox` task. And this is not good because you'll have to reinstall all plugins from debug IDE instance on each recompile